diff --git a/node_modules/moti/build/components/safe-area-view.js b/node_modules/moti/build/components/safe-area-view.js
index ed2108d..065fb5c 100644
--- a/node_modules/moti/build/components/safe-area-view.js
+++ b/node_modules/moti/build/components/safe-area-view.js
@@ -1,4 +1,4 @@
-import { SafeAreaView as RSafeAreaView } from 'react-native';
+import { SafeAreaView as RSafeAreaView } from 'react-native-safe-area-context';
 import { motify } from '../core';
 export const SafeAreaView = motify(RSafeAreaView)();
 //# sourceMappingURL=safe-area-view.js.map
diff --git a/node_modules/moti/src/components/safe-area-view.tsx b/node_modules/moti/src/components/safe-area-view.tsx
index 22d3f9a..7a72240 100644
--- a/node_modules/moti/src/components/safe-area-view.tsx
+++ b/node_modules/moti/src/components/safe-area-view.tsx
@@ -1,4 +1,4 @@
-import { SafeAreaView as RSafeAreaView } from 'react-native'
+import { SafeAreaView as RSafeAreaView } from 'react-native-safe-area-context'
 
 import { motify } from '../core'
 
